app/src/main/res/drawable-v21/ic_next_fragment.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<vector xmlns:android="http://schemas.android.com/apk/res/android" android:height="48dp" android:viewportHeight="24.0" android:viewportWidth="24.0" android:width="48dp">
```

Comment:
WARNING: [UnusedResources] The resource `R.drawable.ic_next_fragment` appears to be unused
 Unused resources make applications larger and slow down builds.